src/main/java/org/concordion/api/AbstractCommand.java

```diff
@@ -25,4 +25,9 @@ public boolean isExample() {
 
     public void executeAsExample(CommandCall commandCall, Evaluator evaluator, ResultRecorder resultRecorder) {
     }
+
+    public boolean alwaysRunEvenIfNoNonExampleChildren() {
+        return false;
+    }
+

```

Comment:
How about we use a marker interface, or annotation, on TableRowExampleCommand (eg. `public class TableRowExampleCommand implements Command, CommandExecutesAsExample`)? 
This would remove the need to add an additional method, but with the downside that authors of new commands would need to know about it if they wanted to use the feature.
Or could we just use the existing `isExample()` method?